### PR TITLE
Pop up details

### PR DESCRIPTION
**Pull request by PowerLevel9000**
# Popup details
 - added a javascript dynamics for pop-up card 
 - added different styling for the mobile popup and desktop popup
 - functionality with respective buttons
 
# problems

1.  I can't animate through logic 
2.  card list of project owner names has some issues can't adjust the style type circle
3.  also the last card Iframe height problem in the desktop version 

WaitingForYouToTextMeBackWaitingGIF

I will try to fix those issues asap

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -90,90 +107,98 @@ <h2><a href="mailto:adi8090808766@gmail.com" target="_blank">LET&apos;S CONNECT<
...
-            rel="noopener noreferrer"><button>See Project</button></a>
+          <div class=buttonlink onclick="modal(1)">
+            <button>See Project</button>
+          </div>
         </div>
       </div>
 
       <div class="project one">
-        <img src="projects-img/tribute-project.png" alt="tribute-project" data-aos="fade-right" data-aos-duration="1500">
+        <img title="Project" src="projects-img/tribute-project.png" alt="tribute-project" data-aos="fade-right"
```
Comment (Qoosim) [inline index.html]: - [x] It is required that the projects section should be created dynamically from your js file. And all the previously created code in the index.html should be removed. These projects created on page load. Kindly remove these code and create them dynamically from js file on page load.

project-rqtmt

**Review by Qoosim — CHANGES_REQUESTED** [review]
# *Status: Required Changes* :recycle: 

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @PowerLevel9000 
I am Favour 👩🏽 , your reviewer for this PR.

Great Job implementing the required change suggested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Great design/UI ✔️ 
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PowerLevel9000** [comment]
# Thank you 💪💪💪🙏🙏🙏

ThankYouSoMuchPrabhasGIF


Dear @Qoosim @Favourezeugwa ,

I wanted to express my gratitude for your willingness to take the time to review my code on GitHub. Your feedback and suggestions have been instrumental in improving the quality of my work, and I appreciate your insights and expertise in this area.

It's inspiring to have someone like you, who is knowledgeable and experienced, provide valuable feedback that helps me enhance my coding skills. I truly appreciate the time and effort you put into reviewing my code, and I'm grateful for your constructive criticism.

If there are any areas that I need to work on, please don't hesitate to let me know. I'm open to hearing any suggestions or recommendations you may have. Your feedback is essential to my growth as a developer, and I value it tremendously.

Thank you again for your time and for sharing your expertise with me. I'm looking forward to applying the insights gained from your review to my future projects.

Sincerely,
@PowerLevel9000

**Pull request merged**